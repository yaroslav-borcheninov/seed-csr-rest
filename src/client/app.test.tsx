import App from "./app"

jest.mock("./components/bootstrap", () => () => null)

describe("App", () => {
  it("should render", () => {
    expect(App).toMatchSnapshot()
  })
})
