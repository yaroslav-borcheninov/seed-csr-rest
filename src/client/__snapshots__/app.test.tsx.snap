// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render 1`] = `
<React.Fragment>
  <BrowserRouter>
    <Switch>
      <Redirect
        exact={true}
        from="/"
        push={false}
        to="items"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/items"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/items/:id"
      />
      <Route
        component={[Function]}
      />
    </Switch>
  </BrowserRouter>
  <GlobalStyleComponent
    suppressMultiMountWarning={false}
  />
</React.Fragment>
`;
